js/core/utils/dom.js

```diff
@@ -179,10 +179,18 @@ var setPublicElementWrapper = function(value) {
     elementStrategy = value;
 };
 
+var getElementMaxHeightByWindow = function($element) {
+    var offsetTop = $element.offset().top - $(window).scrollTop(),
+        offsetBottom = $(window).innerHeight() - offsetTop - $element.outerHeight();
+
+    return Math.max(offsetTop, offsetBottom) * 0.9;
+};
+
```

Comment:
Should it be moved to the `utils/size`? We also can replace the same function in the dxDropDownList to remove code duplication

Reply:
Thank you, for feedback. I will improve this.